scripts/generate_dump

```diff
@@ -698,16 +1003,25 @@ main() {
     save_cmd "sysctl -a" "sysctl"
 
     save_ip_info
+
+    save_bridge "fdb show" "fdb"
+    save_bridge "vlan show" "vlan"
+
+    save_frr_info
     save_bgp_info
 
     save_cmd "show interface status -d all" "interface.status"
-    save_cmd "show interface counters -d all" "interface.counters"
     save_cmd "show interface transceiver presence" "interface.xcvrs.presence"
     save_cmd "show interface transceiver eeprom --dom" "interface.xcvrs.eeprom"
+    save_cmd "show ip interface" "ip.interface"
```

Comment:
use "save_cmd_all_ns" for "show ip interface".

Reply:
Done

Reply:
Done